packages/core/test/unit/mutants/find-mutant-test-coverage.spec.ts

```diff
@@ -146,10 +148,26 @@ describe(sut.name, () => {
       const result = act(dryRunResult, mutants);
 
       // Assert
-      const expected: MutantTestCoverage[] = [{ ...mutant, estimatedNetTime: 0, static: true, coveredBy: undefined }];
+      const expected: MutantTestCoverage[] = [{ ...mutant, estimatedNetTime: 0, static: true, coveredBy: undefined, hitCount: 1 }];
       expect(result).deep.eq(expected);
     });
 
+    it('should calculate the hitCount based on total hits (perTest and static)', () => {
+      // Arrange
+      const mutant = factory.mutant({ id: '1' });
+      const mutants = [mutant];
+      const dryRunResult = factory.completeDryRunResult({
+        tests: [factory.successTestResult({ id: 'spec1', timeSpentMs: 0 })],
+        mutantCoverage: { static: { 1: 1 }, perTest: { 1: { 1: 2, 2: 100 }, 2: { 2: 100 }, 3: { 1: 3 } } },
```

Comment:
I would find this more readable:
```
mutantCoverage: { 
    static: { 1: 1 },
    perTest: { 
        1: { 1: 2, 2: 100 },
        2: { 2: 100 },
        3: { 1: 3 } 
    }
},
```

In the current form, it's a lot of similar looking numbers on one line which can be confusing

Reply:
Sorry, no can do 😟 Prettier doesn't allow it